pkg/sosreport/manifest.json

```diff
@@ -3,7 +3,7 @@
 
     "tools": {
         "index": {
-            "label": "Diagnostic report"
+            "label": "Diagnostic reports"
```

Comment:
@petervo The UI looks OK for me.  As I know, The manifest.json is just for UI layout.